doc/API.md

```diff
@@ -29,6 +30,21 @@ Note that all addresses are bech32-encoded *version 0* native Segwit `scriptPubK
 
 ## General
 
+### `help`
+
+Display all available commands.
+
+#### Response
+
+| Field         | Type          | 
+| ------------- | ------------- | 
+| `name`        | string        | 
+| `parameters`  | string array  | 
+| `description` | string        | 
+
+Optional Parameters are enclosed in `[` `]`
```

Comment:
If i remember correctly this isn't the format of the response? It is something like:
```
"commands": [
    {},
    {}
]
```

Therefore, an array of objects. To follow what we use elsewhere you can say:

#### Response


| Field               | Type          | Description |
| ------------------ | ------------- |  ---------------- |
| "commands" | array        | Array of [help resource](help_resource) |

#### Help resource

| Field         | Type          | Description |
| ------------- | ------------- |  ---------------- |
| `name`        | string        |  The name of the command |
| `parameters`  | array  | The command parameters as strings, optional ones are enclosed in brackets (ie `[optional argument]`) |
| `description` | string        |  A description of the command |

Reply:
Yeah, this looks perfect

Reply:
Pushed the required changes